README.md

```diff
@@ -125,6 +125,11 @@ To run the test suite:
 $ npm test
 ```
 
+To run benchmarks:
+```shell
+$ npm run benchmarks
+```
+
```

Comment:
A note about how to view the results would be helpful since this runs without output